client/src/index.js

```diff
@@ -1,17 +1,5 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
 import App from "./App";
-import reportWebVitals from "./reportWebVitals";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
```

Comment:
@ShepLT1 is there a reason you removed strictMode from this file? seems like we should leave it in.